src/Microsoft.IdentityModel.Tokens.Saml/Saml/Exceptions/SamlSecurityTokenException.cs

```diff
@@ -17,15 +17,17 @@ public class SamlSecurityTokenException : SecurityTokenException
         /// </summary>
         public SamlSecurityTokenException()
             : base()
-        {}
+        {
```

Comment:
I would move this back as they were, they like to have as few lines used as possible. Less noise on the PR as well.

Reply:
Reverted this as part of the changes I think my editor made that Brent pointed out